src/app/(social)/_components/News.tsx

```
@@ -0,0 +1,42 @@
+      <h3 className="text-[15px]/[20px] tracking-[0.5px] font-bold text-white">
+        Latest News
+      </h3>
+      <Col className="gap-3.75">
+        {news.map((item) => (
+          <Col key={`news-${item.id}`} className="py-2.5">
+            <h4 className="text-base/[25px] tracking-[0.5px] align-middle font-medium">
+              {item.title}
+            </h4>
+            <p
+              className="text-sm/[20px] align-middle font-light line-clamp-2 whitespace-normal"
+              dangerouslySetInnerHTML={{ __html: item.html_content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)